packages/fixtures/fixtures.coffee

```diff
@@ -44,3 +45,25 @@ do ->
 
     'createOrg': (attributes) ->
       Organizations.insert attributes
+
+    'addDataset': (attributes)->
+      dataset = new Dataset()
+      dataset.set(attributes)
+      (Meteor.wrapAsync((callback)->
+        dataset.save ->
+          fileObject = new FS.File()
+          fileObject.name("test")
+          fileObject.owner = attributes.createdById
+          callback()
+          # Inserting the file results in errors like this:
+          # "Error: 55ba38ae835506d61cce6500 does not exist"
```

Comment:
That's because the file object doesn't have any file data. We could load something from the file system and attach it to the object. Or make a separate fixtures method for the raw files, since not all datasets will have them. I don't think we should leave in commented out code.